CrazyCanvas/Source/World/LevelObjectCreator.cpp

```diff
@@ -1043,6 +1123,11 @@ bool LevelObjectCreator::CreatePlayer(
 
 	pECS->AddComponent<NetworkComponent>(weaponEntity, { weaponNetworkUID });
 
+	// ShowerComponent needs to be checked from both server and client in order to trigger either of ResetHealth or ResetServer(Paint)
+	const Timestamp	showerCooldown = Timestamp::Seconds(5.0f);
+	const ShowerComponent showerComponent{ EngineLoop::GetTimeSinceStart() + showerCooldown, showerCooldown };
+	pECS->AddComponent<ShowerComponent>(playerEntity, showerComponent);
+
```

Comment:
Where is the code that gives the player more health?
Why is the timer needed on the client side?